code/graphics/gropengltexture.cpp

```diff
@@ -1198,116 +1198,6 @@ void gr_opengl_set_texture_addressing(int mode)
 	GL_CHECK_FOR_ERRORS("end of set_texture_addressing()");
 }
 
-int opengl_compress_image( ubyte **compressed_data, ubyte *in_data, int width, int height, int alpha, int num_mipmaps )
-{
```

Comment:
As noted in the description, I removed this code because it's pointless. It was only used when a command line option was specified that no one uses anymore. Furthermore, using OpenGL for DDS compression is just wrong. If that functionality is needed then it should be done outside the graphics code using an external library.